### ISSUE TITLE
33.1.1.713 - Android cannot access local files anymore (works in 633)

### ISSUE THREAD

**Issue by PippoApps** [airsdk/Adobe-Runtime-Support#1714]
## Problem Description
Up until 33.1.1.633 my Android app could access local files embedded in application.
Since 33.1.1.713 files cannot be accessed anymore. PErmission event reports: denied, although the correct prompt requests access to files.
Same exact setup works using up to 633.
Did anything change in permissions requests?

## Known Workarounds
Use 33.1.1.633

**Comment by lwmirkk** [comment]
I have this problem too, related: https://github.com/airsdk/Adobe-Runtime-Support/issues/1437#issuecomment-1048806858

**Comment by marchbold** [comment]
Where are the files you are trying to access and how are you attempting to access them? 

There has been significant changes to file access for android applications in the latest Android SDK iterations. Most apps are now sandboxed and cannot access files outside of this location.

**Comment by marchbold** [comment]
Yeah exactly, you should always use `applicationStorageDirectory`. 

`documentDirectory` shouldn't be used on Android instead you should use a share mechanism or "save to camera roll" to share/save assets/media you want the user to access outside your application.

**Comment by pol2095** [comment]
[documentsDirectory](https://help.adobe.com/fr_FR/FlashPlatform/reference/actionscript/3/flash/filesystem/File.html#documentsDirectory) access denied but File.documentsDirectory.resolvePath("Documents"), File.documentsDirectory.resolvePath("DCIM"), File.documentsDirectory.resolvePath("Picture") must working.
